app/views/wishlists/show.html.erb

```diff
@@ -1,7 +1,14 @@
 <div class="container">
   <h2>
     <%= @wishlist.name %>
-    <%= social_share_button_tag("Are you a hero of play? #{@wishlist.name} needs your help!") %>
+
+    <% if Rails.env.production? %>
+        <% url_share = "https://project-playtime-staging.herokuapp.com/wishlists/#{@wishlist.id}" %>
+    <% else %>
+        <% url_share = "https://project-playtime-staging.herokuapp.com/wishlists/#{@wishlist.id}" %>
+    <% end %>
+
+    <%= social_share_button_tag("Are you a hero of play? #{@wishlist.name} needs your help!",:url => url_share,desc: "Are you a hero of play? #{@wishlist.name} needs your help!") %>
```

Comment:
Some style nitpicks:
  * You're using two different hash formats here (`:url => ...` and `desc: ...`); let's stick to the second one (`key: value`).
  * For readability, either add spaces after the commas or break this up into multiple lines.

Reply:
i will take care of it